execute/exectypes/observation.go

```diff
@@ -8,6 +8,13 @@ import (
 	dt "github.com/smartcontractkit/chainlink-ccip/internal/plugincommon/discovery/discoverytypes"
 )
 
+// MessageWithSrcChain is a message wrapper that includes the source chain selector.
+// It is used to facilitate the fetching of DestChainConfig from the src chain FeeQuoter.
+type MessageWithSrcChain struct {
+	Message     cciptypes.Message
+	SourceChain cciptypes.ChainSelector
```

Comment:
Source chain selector is already included in the message header, this is not needed